script/1_0_2current_en_script/Arcueid/Day 3/03_00_ARC03_6.txt

```diff
@@ -41,7 +41,7 @@ Hisui slightly raises her eyebrow.
 "...No way. I don't remember that at all."
 <Page11>
 "I do not think it is so strange that you do not remember.
-Your consciousness could have been clouded by weakness caused by anemia."
+Your consciousness could have been clouded by weakness caused by your anemia."

```

Comment:
Your consciousness could have been clouded by your anemia.

The 'weakness' is extraneous here so this can become quite a simple statement of fact from Hisui.